ivy/functional/backends/jax/statistical.py

```diff
@@ -37,7 +37,22 @@ def max(
     return jnp.max(a=jnp.asarray(x), axis=axis, keepdims=keepdims)
 
 
-@with_unsupported_dtypes({"0.4.14 and below": "bfloat16"}, backend_version)
+@with_unsupported_dtypes(
+    {
+        "0.4.19 and below": (
+            "bfloat16",
+            "int8",
```

Comment:
I think we can just use the `int` and `uint` typesets instead of explicitly specifying all signed and unsigned integer dtypes.
So for e.g. 
```
"bfloat16",
"int",
"uint"
```
would be equivalent to the `tuple` you've specified.
Also, I think ideally `mean` should support integers, are there any errors when passing ints to `mean`?

Reply:
This error has been fixed on [`main`](https://github.com/unifyai/ivy/commit/d038b486dac1b8ad04bfaff90f99a5cae31a6449) now, could you please pull from `main` and try again? Thanks @Aaryan562 😄

Reply:
> I think we can just use the `int` and `uint` typesets instead of explicitly specifying all signed and unsigned integer dtypes. So for e.g.
> 
> ```
> "bfloat16",
> "int",
> "uint"
> ```
> 
> would be equivalent to the `tuple` you've specified. Also, I think ideally `mean` should support integers, are there any errors when passing ints to `mean`?

@vedpatwardhan If I don't  explicitly mention the datatypes then it generated those `int` and `uint` datatypes and gives the `AssertionError: returned dtype = float32, ground-truth returned dtype = int8`

Reply:
> > I think we can just use the `int` and `uint` typesets instead of explicitly specifying all signed and unsigned integer dtypes. So for e.g.
> > ```
> > "bfloat16",
> > "int",
> > "uint"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > would be equivalent to the `tuple` you've specified. Also, I think ideally `mean` should support integers, are there any errors when passing ints to `mean`?
> 
> @vedpatwardhan If I don't explicitly mention the datatypes then it generated those `int` and `uint` datatypes and gives the `AssertionError: returned dtype = float32, ground-truth returned dtype = int8`

Does that mean I need to make sure that the returned dtype should be of `int8` as it is not case of unsupported datatype. Also as discussed even though the python array api docs suggest that for mean we should have floating datatypes we should go beyond that as it is minimum standard.

Reply:
> I think we can just use the `int` and `uint` typesets instead of explicitly specifying all signed and unsigned integer dtypes. So for e.g.
> 
> ```
> "bfloat16",
> "int",
> "uint"
> ```
> 
> would be equivalent to the `tuple` you've specified. Also, I think ideally `mean` should support integers, are there any errors when passing ints to `mean`?

It is `integer` and `unsigned` which is equivalent to all integer and unsigned integers